dev/drivers/scripts/plots/cam/jevs_cam_grid2obs_plots.sh

```diff
@@ -49,8 +50,9 @@ evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
 # Developer Settings
 export envir=prod
 export DATAROOT=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
-export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver_2d
+export COMIN=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/$NET/$evs_ver_2d
```

Comment:
Revert lines 38 and 53 back to original definitions.

Reply:
nevermind -- you told me you know about this one and precip plots and you will fix in next commit.